# Issue on setting up newRelic on Python 3.4
sudo chmod 755 YOUR_FILE.py
